content/en/_index.html

```diff
@@ -36,10 +41,12 @@ <h3 class="section-head">What is Kubeflow?</h3>
 </div>
 
 <section id="pageContent">
-  <h3 class="section-head text-center">Kubeflow Components</h3>
   <div class="container">
+    <h3 class="section-head text-center">
```

Comment:
I would suggest we whethere:
1. Remove this green lines between divs
2. Change it back to blue color.

I don't really like how this green line looks like on the white screen.

WDYT @thesuperzapper ?
![Screenshot 2025-03-30 at 00 39 18](https://github.com/user-attachments/assets/130c1169-3da2-4b2d-b3f7-31d1478e318a)

Reply:
Its technically a "teal" color, but we can make it dark/light blue if you want, let me see what it looks like.

Reply:
@andreyvelich I agree it looks much better with a slightly transparent grey (the darkness of which depends on if its dark/light mode).

Check it out now, and tell me if you like it (might need to do a "hard refresh and clear cache" by holding down on the refresh button while the dev tools panel is open).

- https://deploy-preview-3981--competent-brattain-de2d6d.netlify.app/

Reply:
Yeah, looks great.